repository/userRepository.py

```diff
@@ -0,0 +1,159 @@
+# w plikach typu repository, przechowuje sie metody i zapytania czysto odpowiedzialne za CRUD zapytan bazodanowych
+
+import sqlite3
+
+
+# todo we will be working on database during all operations, there
+# should be open connection in main class and somehow repos and everything should operate on this to
+# avoid opening and closing this everytime
+
+def initiate_db():
+    conn = sqlite3.connect('database/gitfetcherdb.db')
+
+    db = conn.cursor()
+    db.execute('DROP TABLE IF EXISTS users')
+    db.execute("""
+    CREATE TABLE IF NOT EXISTS users ( 
+                login text,
+                id integer,
+                node_id text,
+                avatar_url text,
+                gravatar_id text,
+                url text,
+                html_url text,
+                followers_url text,
+                following_url text,
+                gists_url text,
+                starred_url text,
+                subscriptions_url text,
+                organizations_url text,
+                repos_url text,
+                events_url text,
+                received_events_url text,
+                type text,
+                site_admin text,
+                name text,
+                company text,
+                blog text,
+                location text,
+                email text ,
+                hireable text,
+                bio text,
+                twitter_username text,
+                public_repos integer,
+                public_gists integer,
+                followers integer,
+                following integer,
+                created_at text,
+                updated_at text)
+                     """)
+    print(f"Table users is created")
+
+    conn.commit()
+    conn.close()
+
+
+def get_all_users():
+    conn = sqlite3.connect('database/gitfetcherdb.db')
+    db = conn.cursor()
+    db.execute("""SELECT * FROM users""")
+    return db.fetchall()
+
+
+def get_user_by_name(name):
+    conn = sqlite3.connect('database/gitfetcherdb.db')
+
+    sql = """SELECT * FROM users WHERE login = ?"""
+
+    db = conn.cursor()
+    db.execute(sql, (name,))
+    return db.fetchall()
+
+
+def drop_table(table_name):
```

Comment:
ta metoda wystepuje i w repo contributorow i w repo user. To sie nazywa duplikacja kodu i jest to cos bardzo zlego w programowaniu. 
Proponuje zrobic osobny plik w folderze `utils` i nazwac `utils.py` i tam zrobic ta metode. Jak bedziemy potrzebowac to wtedy ja importujemy.